README.md

```diff
@@ -54,6 +62,27 @@ Setting up logrotate for additional Nginx logs, with postrotate script (assuming
         - copytruncate
       scripts:
         postrotate: "[ -s /run/nginx.pid ] && kill -USR1 `cat /run/nginx.pid`"
+    - name: access_logs
+      path: /etc/foo.log
```

Comment:
@JoeWoodward is `foo.log` in here deliberately? It looks like a copy-n-pasted example

Reply:
yeah, it's just an example